app/src/main/java/lt/vilnius/tvarkau/NewProblemActivity.java

```diff
@@ -230,8 +236,25 @@ protected void onActivityResult(int requestCode, int resultCode, Intent data) {
                     break;
                 case REQUEST_PLACE_PICKER:
                     Place place = PlacePicker.getPlace(this, data);
-                    mAddProblemLocation.setText(place.getName());
-                    locationCords = place.getLatLng();
+                    LatLng latLng = place.getLatLng();
+
+                    Geocoder geocoder = new Geocoder(getApplication(), Locale.getDefault());
```

Comment:
If you use default locale use `new Geocoder(Context)` constructor. In this way geocoder will use default locale.